supervisord.conf

```diff
@@ -22,7 +70,7 @@ autorestart = true
 user = www-data
 directory = /var/lib/graphite/webapp
 environment = PYTHONPATH='/var/lib/graphite/webapp',DJANGO_SETTINGS_MODULE='graphite.settings'
-command = /usr/bin/gunicorn graphite.wsgi:application -b127.0.0.1:8000
+command = /usr/bin/gunicorn --workers=8 graphite.wsgi:application -b127.0.0.1:8000
```

Comment:
so we use a more performant carbon implementation now...
also we let it run with 8 workers now (instead of 1 previously?) and here we let graphite use 8 workers as well, right?

do all of these changes combined mean that the minimum cpu requirements for running this image have increased?
or could i still run this on a machine with just 2 physical cpus?
(asking because the default machine size we use for this only has 2 cpus)

is there any chance to make this "parallelism" parameter configurable on container start via a parameter?
(or by detecting the number of available CPUs?).

Reply:
No, the same 2-CPU instances can still be used.  If they were saturating their disk layer, then spreading the requests over more threads won't change that fact.  And if they were saturating because of the single-threaded nature of the old carbon-cache, now they will see a boost in performance.

> is there any chance to make this "parallelism" parameter configurable on container start via a parameter?

That is beyond my docker skills :-(